docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -2,6 +2,18 @@
 Query management properties
 ===========================
 
+``query.execution-policy``
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:**
```

Comment:
Missing the info about the type, @mosabua do you know it or is there someone else who you can point me to?

Reply:
Should just be a string type: https://trino.io/docs/current/admin/properties.html#string

Reply:
@jhlodin thanks for the clarification!